src/sass/components/_button.scss

```diff
@@ -12,6 +12,16 @@
     text-decoration: none;
   }
 }
+.btn-main-more {
+  background-color: #2a2a2a;
+  color: #ffffff;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 600;
+  padding: 10px 30px;
+  margin: 0 75px 0 75px;
```

Comment:
You may use here shorthand, I mean just `0 75px`.

Reply:
Fixed in new commit